docs/running/amazon.rst

```diff
@@ -22,7 +22,7 @@ The :ref:`StaticProvisioning` section explains how a static cluster (one that
 won't automatically change in size) can be created and provisioned (grown, shrunk, destroyed, etc.).
 
 Currently, it's recommended that users do not run their scripts from system folders (``/var``, ``/``, or ``/tmp`` for example.).
-
+It's recommended to create a new folder such as ``/tmp/working`` as the working directory.
```

Comment:
Please explain why it is recommended.  Recommendations that don't teach people is for robots ;-)

 Not necessarily a new folder, because it might be a venv or one where your app is being developed.  Mentioning /var is almost weird, as I don't think many people would do this.   The fact that it  /tmp is shared by other processes means attempts to copy other files in /tmp is the real problem.